src/haddock/modules/sampling/rigidbody/cns/rigidbody.cns

```diff
@@ -23,7 +23,6 @@ end if
 
 evaluate ($saprotocol.crossdock=$crossdock)
 evaluate ($saprotocol.randorien=$randorien)
-evaluate ($saprotocol.rigidmini=$rigidmini)
```

Comment:
This has no use anymore, right?

Reply:
`ridigmini` has indeed no use anymore. You simply don't call that module if you want to skip it.  One example of this is the `refine-complex` example.

But what if we want to dock during the flexible refinement stage (we did it in the past for some special cases)?  We might need in that case a `randorient` module after `topoaa`

Reply:
allright! seems quite specific, but now more then ever we have the "flexibility" to do it! 😄